main.cmake

```diff
@@ -44,6 +46,18 @@ option(MENDER_ARTIFACT_GZIP_COMPRESSION "Enable GZIP compression support when do
 option(MENDER_ARTIFACT_LZMA_COMPRESSION "Enable LZMA compression support when downloading and extracting Artifacts (Default: ON)" ON)
 option(MENDER_ARTIFACT_ZSTD_COMPRESSION "Enable Zstd compression support when downloading and extracting Artifacts (Default: ON)" ON)
 
+# Coverage may fail when building tests.
```

Comment:
Q: Should this be a `message` perhaps? :thinking:

Reply:
Maybe a warning?

Reply:
Yes, maybe even :)